packages/framework/presence/src/test/presenceManager.spec.ts

```diff
@@ -373,30 +443,33 @@ describe("Presence", () => {
 					}
 
 					describe("and has their connection removed", () => {
-						it("is announced via `attendeeDisconnected`", () => {
```

Comment:
I just realized the grammar of what these cases are saying after rereading the PR description. Thank for putting the test list in there. If you read the "sentence" down you get:
  PresenceManager , when connected, attendee that is already known and has their connection removed...
In that situation "their" refers to "attendee". But you mean to refer to this local client.

Reply:
added a seperate describe block to make 
```
Presence
  PresenceManager
    when connected
      attendee
        that is already known
          when local client disconnects
             - updates stale attendee status to 'Disconnected' after 30s delay upon reconnection
             - does not update stale attendee status if local client does not reconnect
             - does not update stale attendee status if local client reconnection lasts less than 30s
             - does not update attendee status to 'Disconnected' if stale attendee becomes active
             - updates stale attendee status to 'Disconnected' only 30s after most recent reconnection